DiscordRichPresence/DiscordRichPresence.cpp

```diff
@@ -93,38 +93,45 @@ int init()
 
     LoadSettingsFile();
 	
-	g_presenceInfo.InitializeDiscordRPC();
+	if (g_pluginSettings.SendWhenPausedOrStopped)
+	{
+		g_presenceInfo.InitializeDiscordRPC();
 
-    ReportIdleStatus();
+    	ReportIdleStatus();
+	}
 
     return 0;
 }
 
 void ReportIdleStatus()
-{
-    if (!g_presenceInfo.HasDiscordModuleLoaded())
+{    
+	if (g_pluginSettings.ApplicationID == "0")
```

Comment:
weird formatting

Reply:
Eesh, indeed. 

VS Code isn't formatting and I am doing it by hand.  Missed this.